taui/src/components/with-authenticator.js

```diff
@@ -27,7 +28,7 @@ function customAuthErrorMessageMap (error) {
   }
 
   if (customMessages.hasOwnProperty(error)) {
-    return message(customMessages[error])
+    return <Translation>{(t) => t(message(error, customMessages[error]))}</Translation>
```

Comment:
Nice use of `<Translation>` -- that's a great way to inject translation into just this one spot. However, I think we need to remove `message` here rather than preserve it -- I was surprised to see it, and it looks like this was my oversight when switching to `react-i18next` from `woonerf`. Oops (and sorry for leading you astray). I also found a few other instances where there are some vestigial `message` usages lying around and made #406 to get rid of them.